src/components/util/CodelabCard/index.js

```diff
@@ -21,52 +23,63 @@ const CardComponent = ({
 }) => {
   const classes = useStyles();
   const [logoPath, setLogoPath] = React.useState("");
+  
   React.useEffect(() => {
     setLogoPath(org);
   }, [org]);
 
+  dayjs.extend(relativeTime);
+  const timeAgo = (date) => {
+    const dateSplit = date.split(" ");
+    const months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'June', 'July', 'Aug', 'Sept', 'Oct', 'Nov', 'Dec'];
+    const monthFromIdx = months.indexOf(dateSplit[1])+1;
+    const answer = dayjs().to(dayjs(dateSplit[2] + '-' + monthFromIdx + '-' + dateSplit[0]));
```

Comment:
why don't you use javascript template literals?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
I can surely use it but I was habituated with concatenation. If it's poor practice, then I can change with template literal.

Reply:
https://stackoverflow.com/questions/27565056/es6-template-literals-vs-concatenated-strings

I think it is best to use template literals.

Reply:
@Niweera, I will keep it in mind from now onwards. I have replaced it with a template literal. I request you to review it.